typescript/tsconfig-model.json

```diff
@@ -0,0 +1,9 @@
+{
+  "__license": "See LICENSE.md for details.",
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2020",
+    "declaration": true,
+    "outDir": "dist"
+  },
+}
```

Comment:
No newline at end of file. Are you using `revng check-conventions`?

Reply:
There is a bug, adding fix to rcc PR

Reply:
The PR went in before that.

Reply:
https://github.com/revng/revng/pull/172